keps/sig-auth/740-service-account-external-signing/kep.yaml

```diff
@@ -0,0 +1,21 @@
+title: Support external signing of service account keys
+kep-number: 740
+authors:
+  - "@micahhausler"
+owning-sig: sig-auth
+participating-sigs: []
+reviewers:
+  - "@mikedanese"
+  - "@liggit"
+  - "@tallclair"
+approvers:
+  - "@mikedanese"
+  - "@liggit"
+  - "@tallclair"
+editor: '@micahhausler'
+creation-date: 2019-01-16
+last-updated: 2019-05-17
+status: implementable
```

Comment:
This is incorrect as #1564 never made it.  I have opened #3653 to fix this.

Reply:
this just copied the existing implementable status from https://github.com/kubernetes/enhancements/pull/2208/files#diff-d3f596ee0bc64d6a9a5d8083691e407607119b7b60b2feecee69d9251114adf4L18

it sounds like it was maybe unintentional, but it like the KEP had `implementable` status on initial merge - https://github.com/kubernetes/enhancements/pull/704/files#diff-b8d51ea5e342526a09d4c7a6e6323a77c3a9d05ddca0c1dd0d52dc9f87aa4291R18

Reply:
Ah, let me fix up my description.